samples/openvx/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 2.8.9)
+
```

Comment:
I see, but CMakeLists.txt should be as much self-contained and clear as possible, and these code lines definitely improve it